includes/overlords/contribs.php

```diff
@@ -200,7 +269,9 @@ public static function display_contribs($mode, $id, $branch = false, $sort = fal
 						)
 					),
 
-					'WHERE'		=> ((is_array($id) && sizeof($id)) ? phpbb::$db->sql_in_set('cic.category_id', array_map('intval', $id)) : 'cic.category_id = ' . (int) $id) . '
+					// If multiple categories, use the stripped list. If it's just the single hidden category, that's okay
+					// as presumably someone has gone looking for the hidden category, or it has been linked to, etc.
+					'WHERE'		=> ((is_array($visible_category_ids) && sizeof($visible_category_ids)) ? phpbb::$db->sql_in_set('cic.category_id', array_map('intval', $visible_category_ids)) : 'cic.category_id = ' . (int) $id) . '
```

Comment:
Please use `count` instead of `sizeof`. Also, now `$id` is an array but being cast to an integer? How does that work?

Seems to me like $id can be either an int or an array so that needs to be checked above, before we start treating it as an array by default. And also it needs to be checked here like it was in the old code before we assume it is an integer.